esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
```

Comment:
It could be added then, if necessary, avoiding the cost (size and speed) of adding it earlier "just in case".